assets/javascripts/audit_log.js

```diff
@@ -47,18 +49,34 @@ function loadAuditLogTable ()
             render: function ( data, type, row ) {
                 if (type === 'display')
                     // I want to have a link to events for cases when one wants to share interesing event
-                    return '<a href="' + audit_url + '?eventid=' + row.id + '">' + jQuery.timeago(data + " UTC") + '</a>';
+                    return '<a href="' + audit_url + '?eventid=' + row.id + '" title=' + data + '>' + jQuery.timeago(data + " UTC") + '</a>';
                 else
                     return data;
             }
         },
+        {
+            targets: 2,
+            render: function ( data, type, row ) {
+                // the connection info is not so important to warrant such retail space
+                if (type === 'display' && data) {
+                    return '<span id="audit_event_data" title="' + data + '">' + data.substr( 0, 8 ) + (data.length > 8 ? '…</span>' : '</span>');
+                }
+                else {
+                    return data;
+                }
+            }
+        },
         { 
             targets: 4,
             render: function ( data, type, row ) {
-                // Limit length of displayed event data, expand on click
```

Comment:
this 'expand to click' would btw be a really good idea. Because I often want to duplicate the POST data of a ISO post, but extracting this from a tooltip is tough